client/plots/wsiviewer/WSIViewer.ts

```diff
@@ -205,6 +207,27 @@ export default class WSIViewer {
 		}
 		return layers
 	}
+
+	private renderMetadata(holder: any, layers: Array<TileLayer<Zoomify>>, settings: Settings) {
```

Comment:
thanks. please use client/dom/table2col.js to display this 2-column table

Reply:
Done.